File: app/src/main/java/net/marksheehan/doomtowndeckbuilder/CardModel.java
package net.marksheehan.doomtowndeckbuilder;

import com.google.gson.annotations.SerializedName;

public class CardModel
{
    @SerializedName("last-modified")
    public String lastModified;

    public String code;
    public String title;
    public String type;

    @SerializedName("type_code")
    public String typeCode;

    public String suit;

    public String keywords;
    public String text;

    public long cost;

    public String gang;
    @SerializedName("gang_code")

    public String gang_code;
    public String gang_letter;

    public long number;
    public long quantity;
    public long rank;
    public long upkeep;
    public long bullets;
    public long influence;
    public long control;
    public long wealth;

    public String url;
    public String imagesrc;
}

File: app/src/main/java/net/marksheehan/doomtowndeckbuilder/CardModel.kt
package net.marksheehan.doomtowndeckbuilder

import com.google.gson.annotations.SerializedName

class CardModel {
    @SerializedName("last-modified")
    var lastModified: String? = null

    var code: String? = null
    var title: String? = null
    var type: String? = null

    @SerializedName("type_code")
    var typeCode: String? = null

    var suit: String? = null

    var keywords: String? = null
    var text: String? = null

    var cost: Long = 0

    var gang: String? = null
    @SerializedName("gang_code")
    var gang_code: String? = null
    var gang_letter: String? = null

    var number: Long = 0
    var quantity: Long = 0
    var rank: Long = 0
    var upkeep: Long = 0
    var bullets: Long = 0
    var influence: Long = 0
    var control: Long = 0
    var wealth: Long = 0

    var url: String? = null
    var imagesrc: String? = null
}
